apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/blocks/Typography/Variant/Variant.tsx

```
@@ -93,107 +118,125 @@ export function Variant(): ReactElement {
-      value={selectedBlock?.variant ?? TypographyVariant.body2}
-      onChange={handleChange}
-      options={options}
-      testId="Variant"
-    />
+    <>
+      <ToggleButtonGroup
+        value={selectedBlock?.variant ?? TypographyVariant.body2}
+        onChange={handleChange}
+        options={options}
+        testId="Variant"
+        children={editFontThemeButton}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)